pkg/controller/nodeconfigpod/sync_configmaps.go

```diff
@@ -24,6 +24,10 @@ func (ncpc *Controller) makeConfigMap(ctx context.Context, pod *corev1.Pod) (*co
 		return nil, nil
 	}
 
+	if !controllerhelpers.IsScyllaPod(pod) {
```

Comment:
how does this relate to this PR?

Reply:
Operator tried to create tuning CM for cleanup job, this fixes it.